src/mumble/CoreAudio.cpp

```diff
@@ -37,7 +37,7 @@ class CoreAudioInit : public DeferInit {
 		CoreAudioInputRegistrar *cairReg;
 		CoreAudioOutputRegistrar *caorReg;
 	public:
-		CoreAudioInit() : cairReg(NULL), caorReg(NULL) {}
+		CoreAudioInit() : cairReg(nullptr), caorReg(nullptr) {}
```

Comment:
While I do appreciate that you are modernizing the code-base on-the-fly, these kind of changes should be in their own commit. (A single commit should only deal with one kind of change).
This facilitates reviews and makes commits more easily understandable if viewed sometime in the future.

Reply:
Roger that :).